src/makeCalculator.js

```diff
@@ -5,6 +5,41 @@
  */
 function makeCalculator() {
   // write code here
+  return {
```

Comment:
[CODE KNOWLEDGE] - When creating methods in the object, you don't need to use the function keyword; use the method shorthand instead. For example, use `add(number) { ... },` instead of `add: function(calculator, number) { ... },`.